libs/wire-api/src/Wire/API/User/Orphans.hs

```diff
@@ -51,9 +53,17 @@ instance ToSchema CountryCode
 
 -- | The options to use for schema generation. Must match the options used
 -- for 'ToJSON' instances elsewhere.
+--
+-- FUTUREWORK: This should be removed once the saml2-web-sso types are updated to remove their prefixes.
```

Comment:
Can we add https://wearezeta.atlassian.net/browse/WPB-3972 to the comment?

Reply:
Yup yup